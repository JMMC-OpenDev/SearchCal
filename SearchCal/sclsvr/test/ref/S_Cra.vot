version https://git-lfs.github.com/spec/v1
oid sha256:c283a5ee6396617c0e476d082c9ef65468692d0aca05fe5f2162072135b6c197
size 210440
