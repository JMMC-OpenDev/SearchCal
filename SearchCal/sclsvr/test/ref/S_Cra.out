version https://git-lfs.github.com/spec/v1
oid sha256:be03943f5003cc1456ff3eefb31f297387c05148f969e7e62cf9a4b323825445
size 281029
